modules/administration/pages/tshoot-osadjabberd.adoc

```diff
@@ -23,18 +23,25 @@ If more detailed instructions are required, put them in a "Resolving" procedure:
 
 == Open File Count Exceeded
 
-The number of maximum files that a jabber user can open is lower than the number of connected clients.
-OSAD clients cannot contact the SUSE Manager Server, and jabberd requires long periods of time to respond on port 5222.
-Each client requires one permanently open TCP connection and each connection requires one file handler.
+If you receive an ``open file count exceeded`` error, it could be because the maximum number of files that jabber can open is lower than the number of connected OSAD clients.
 
-To resolve OSAD connection problems, edit such lines in [path]``/etc/security/limits.conf``:
+If this occurs, OSAD clients cannot contact the SUSE Manager Server, and jabberd will take an excessive amount of time to respond on port 5222.
+
+[[NOTE]]
+====
+This fix is only required if you have more than 8192 clients connected using OSAD.
+In this case, we recommend you consider using Salt clients instead.
+For more information about tuning large scale installations, see xref:salt:large-scale.adoc[].
+====
+
+You can increase the number of files available to jabber by editing these lines [path]``/etc/security/limits.conf``:
```

Comment:
@mcalmer is this relevant after `jabberd` is on `systemd`? Isn't this controlled by `LimitNOFILE` in the systemd unit?

See https://build.opensuse.org/package/view_file/systemsmanagement:Uyuni:Stable/jabberd/jabberd-services.patch?expand=1 line 21

Reply:
Right, I think this is not needed anymore. We need to describe how to "overwrite" systemd service files instead.

Reply:
Is this documented somewhere?

Reply:
I hope standard systemd docs (man page or similar) . I never did it so I only heard that it is possible

Reply:
Hmm

https://unix.stackexchange.com/questions/398540/how-to-override-systemd-unit-file-settings

stackoverflow say there is even a command doing it? I never tried it :-)

Reply:
OK. With the systemd method, is there still a way to set soft and hard limits? Or is it just a hard limit?

Reply:
Both is possible. See `man systemd.exec`

```
LimitCPU=, LimitFSIZE=, LimitDATA=, LimitSTACK=, LimitCORE=, LimitRSS=, LimitNOFILE=, LimitAS=, LimitNPROC=, LimitMEMLOCK=, LimitLOCKS=, LimitSIGPENDING=, LimitMSGQUEUE=,
       LimitNICE=, LimitRTPRIO=, LimitRTTIME=
           Set soft and hard limits on various resources for executed processes. See setrlimit(2) for details on the resource limit concept. Resource limits may be specified in two
           formats: either as single value to set a specific soft and hard limit to the same value, or as colon-separated pair soft:hard to set both limits individually (e.g.
           "LimitAS=4G:16G"). 
```